src/event.js

```diff
@@ -610,8 +610,7 @@ jQuery.event = {
 			event,
 			{
 				type: type,
-				isSimulated: true,
-				originalEvent: {}
```

Comment:
> With my proposal it would be a native event.

Yes, but wouldn't that allow for this...

> with a change handler calling event.stopPropagation() actually stopping propagation on the original click event.